### PR TITLE
[New mob]Skeleton creeper

### PR DESCRIPTION
**Pull request by ghost**
This bone-only creeper has a smaller explosion than ordinary creepers
However, when it explodes, it will scatter several bones and bite into the surrounding mobs

I tried to make what Drulkus proposed

### CODE REVIEW

**Comment by Tamaized** [comment]
Linking the discord convo since i went and dug it up
https://discordapp.com/channels/313006291012288521/313006338819096579/511321063251247135

**Comment by Tamaized** [comment]
I noticed a lot of different formatting with the curly braces, try to keep it the same as the rest of the codebase.

```
(...) {

}
```
vs
```
(...)
{

}
```

**Comment by ghost** [comment]
I fix it :)

File: src/main/java/twilightforest/entity/ai/EntityAITFSkeletonCreeperSwell.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package twilightforest.entity.ai;
+
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.ai.EntityAIBase;
+import twilightforest.entity.EntityTFSkeletonCreeper;
+
+public class EntityAITFSkeletonCreeperSwell extends EntityAIBase
+{
```
Comment (Tamaized) [inline src/main/java/twilightforest/entity/ai/EntityAITFSkeletonCreeperSwell.java]: Missed just one file on the formatting :stuck_out_tongue_winking_eye:

**Comment by ghost** [comment]
fixed! :)

**Comment by Tamaized** [comment]
Ah i should have explained better, it was the entire file itself not just the one spot

**Review by Tamaized — APPROVED** [review]

**Review by Drullkus — APPROVED** [review]

**Comment by Tamaized** [comment]
I think this feature still requires discussion between the team. But if we approve it eventually I'll let you know and we can retarget this for whichever the main branch will be at the time.

**Comment by Tamaized** [comment]
Closing as bagu has a new account.

**Pull request closed without merging**